microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -103,6 +103,15 @@ You can delete templates stored on the device and clear the cache data. Go to `e
 You can use the title of your collection to find relevant Pinterest Topic pages when searching in Microsoft Bing. Go to `edge://settings/privacy`. Turn on the **Show suggestions from Pinterest in Collections** setting. Microsoft Edge does not send data about your collections to Pinterest. You can remove the suggestions and stop searches for Pinterest Topic pages. Go to `edge://settings/privacy` and turn off the **Show suggestions from Pinterest in Collections** setting.
 Collections aren't available when using **InPrivate** or **Guest** modes.
 
+<!-- ====================================================================== -->
+## Compose
+
+In the Bing side pane, Compose uses AI to write for you starting from a short prompt and "tone", "format", and "length" selections. Text generated by Compose can be copied or inserted into the text page directly. To access Compose, click on the blue Bing icon in the right hand pane > Compose (requires Bing waitlist approval). Text and selections inputed by the user is sent to an internal Microsoft Large Language Model (LLM).
```

Comment:
```suggestion
In the Bing side pane, Compose uses AI to write for you starting from a short prompt and **Tone**, **Format**, and **Length** selections. Text generated by Compose can be copied or inserted into the text page directly. To access Compose, click the blue Bing icon in the right hand pane > **Compose** (requires Bing waitlist approval). Text and selections that are entered by the user are sent to an internal Microsoft Large Language Model (LLM).
```
https://www.ahdictionary.com/word/search.html?q=inputted - found w/ 2 t's.

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/that-vs-which -- "Include _that_ even if the sentence is clear without it. It helps to clarify the sentence structure."

Reply:
```suggestion
In the Bing side pane, Compose uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections. Text that's generated by Compose can be copied or inserted into the text page directly. To access Compose, click the blue Bing icon in the right hand pane > **Compose** (requires Bing waitlist approval). Text and selections that you enter are sent to an internal Microsoft Large Language Model (LLM).
```

Just a correction to @mikehoffms's suggestion (getting rid of extra 'are'). Otherwise I agree with it.

Reply:
i edited my Suggested Change to get rid of extra "are" in: "that are you enter are sent"

Reply:
@majenny - pls Resolve (close) this comment, it's been incorp'd at latest https://github.com/MicrosoftDocs/edge-developer/pull/2574/files > Compose section.

gh rendered section:
https://github.com/MicrosoftDocs/edge-developer/blob/user/majenny/ComposePrivacy/microsoft-edge/privacy-whitepaper/index.md#compose